stylelint.config.js

```diff
@@ -4,7 +4,5 @@ module.exports = {
     'function-comma-newline-after': null,
     'function-parentheses-newline-inside': null,
     'max-empty-lines': null,
```

Comment:
Nitpick: shouldn't the trailing comma be removed?

Reply:
The AirBnB ESLint rules that we use for ES2015 require a comma on the last line in lists:

http://airbnb.io/javascript/#commas--dangling

Reply:
Thanks for clarifying.